modules/SofaMiscFem/TriangularFEMForceField.h

```diff
@@ -255,7 +257,16 @@ class TriangularFEMForceField : public core::behavior::ForceField<DataTypes>
     void setDamping(Real val) { f_damping.setValue(val); }
     int  getMethod() { return method; }
     void setMethod(int val) { method = val; }
-
+    void setMethod(std::string methodName) {
```

Comment:
As the setMethod can fails, shouldn't it be more adequate to returns a boolean indicating it has succeeded or not ?